src/pages/leeszaal/[readingRoomSlug]/index.tsx

```diff
@@ -57,6 +58,8 @@ import { VisitorLayout } from 'modules/visitors';
 
 const ReadingRoomPage: NextPage = () => {
 	const { t } = useTranslation();
+	const router = useRouter();
+	const { readingRoomSlug } = router.query;
```

Comment:
Shouldn't this be `router.path`?

Reply:
no router.query contains both queryParams and route params:
https://nextjs.org/docs/routing/dynamic-routes

![image](https://user-images.githubusercontent.com/1710840/159714394-2e296b34-5038-4a10-9b11-48a0a03c6af7.png)